bears/js/JSHintBear.py

```diff
@@ -140,8 +140,8 @@ def generate_config(filename, file,
             Allows "non-breaking whitespace characters".
         :param allow_constructor_functions:
             Allows the use of constructor functions..
-        :param prohibit_undefined:
-            This option prohibits the use of explicitly undeclared variables.
+        :param allow_undefined_variables:
+            Allows the use of explicitly undeclared variables.
```

Comment:
how can you allow the use of explicitly UNDECLARED variabiles? im so confused about this property